Lib/idlelib/idle_test/htest.py

```diff
@@ -209,7 +208,7 @@ def _wrapper(parent):  # htest #
 _linenumbers_drag_scrolling_spec = {
     'file': 'sidebar',
     'kwds': {},
-    'msg': textwrap.dedent("""\
+    'msg': """\
```

Comment:
I don't like having '.dedent moved from here to the end 16 lines down.

In any case, I want both idlelib changes removed from this PR to avoid gratuitous differences between versions.